packages/search-service/src/resolver.ts

```diff
@@ -0,0 +1,17 @@
+import { SearchIndexHelper } from './helpers';
+
+export const SearchResolver = {
+  Query: {
+    search ( root: any, args: any, ctx: any ) {
+      return SearchIndexHelper.search( args ).then( res => res);
```

Comment:
`.then` not needed as it isn't doing anything.

```suggestion
      return SearchIndexHelper.search( args );
```